Userland/Libraries/LibJS/Runtime/Temporal/PlainTime.cpp

```diff
@@ -33,8 +34,38 @@ void PlainTime::visit_edges(Visitor& visitor)
     visitor.visit(&m_calendar);
 }
 
+// 4.5.4 RegulateTime ( hour, minute, second, millisecond, microsecond, nanosecond, overflow ), https://tc39.es/proposal-temporal/#sec-temporal-regulatetime
+Optional<TemporalTime> regulate_time(GlobalObject& global_object, double hour, double minute, double second, double millisecond, double microsecond, double nanosecond, StringView overflow)
+{
+    auto& vm = global_object.vm();
+
+    // 1. Assert: hour, minute, second, millisecond, microsecond and nanosecond are integers.
```

Comment:
Since these are doubles, are we actually sure they're integers at this point?

Reply:
Great question, and the answer is no. These fields are converted using `ToIntegerOrInfinity`, so they can be infinity when they enter this function, and will only be rejected later - there's an open spec issue about this: https://github.com/tc39/proposal-temporal/issues/1672

I'll add a comment :+1: